src/control-plane/control-plane.ts

```diff
@@ -6,30 +6,39 @@ import { EventBus } from 'aws-cdk-lib/aws-events';
 import { NagSuppressions } from 'cdk-nag';
 import { Construct } from 'constructs';
 import { IAuth } from './auth';
+import { IBilling } from './billing/billing-interface';
```

Comment:
Would it make sense to roll up everything in a barrel and export * from billing?
```suggestion
import { IBilling, BillingProvider } from './billing';
```

Reply:
Updated!